src/main/resources/assets/eln2/lang/en_us.json

```diff
@@ -29,13 +29,13 @@
     "plotter.eln2.energy": "Energy",
     "plotter.eln2.power": "Power",
     "plotter.eln2.mode": "Mode",
-
     "waila.eln2.voltage": "Voltage",
     "waila.eln2.current": "Current",
     "waila.eln2.resistance": "Resistance",
     "waila.eln2.capacitance": "Capacitance",
     "waila.eln2.inductance": "Inductance",
     "waila.eln2.energy": "Energy",
     "waila.eln2.power": "Power",
-    "waila.eln2.mode": "Mode"
+    "waila.eln2.mode": "Mode",
+    "gui.eln2.save": "Save"
```

Comment:
I'd add some whitespace between these, as I've been trying to space them out when they are different components to be translated (might make it easier on our translators)